_example/hello-world/Makefile

```diff
@@ -10,16 +10,16 @@ pack:
 
 dbuild:
 	@docker run --rm \
-		-u ${shell id -u}:${shell id -g} \
+		-v $(PWD):/usr/src/hello-world \
```

Comment:
Your proposition is very interesting here, however, we need to make it more generic. As more examples will come in the near future :), I need to be sure that we can repeat the same folder structure and files for any of them.
Here is what I suggest, and if it is ok the submit changes accordingly:
```
		-v ${GOPATH}:/go \
		-v $(PWD):/usr/src/current \
		-w /usr/src/current \
```
Here I've reordered your proposition and changed `hello-world` by `current` (or any other meaningful name) in order to be able to use the same Makefile in future examples. 
In fact, as the name of the final output is fixed (handler.so), the name of the folder can be anything generic.